### ISSUE TITLE
help on flattening

### ISSUE THREAD

**Issue by uat230214** [pdfcpu/pdfcpu#563]
Is the on the oven forms functionallity able to flatten, in the sense that https://github.com/cityssm/pdfFlattener does?
(sorry but my pdf knowledge is just user one... I am looking for a way to convert pdf content to just image with good compression... now I do it using imagemagick convert plus pdfflattener but pdfflattener does not support command line ...)

Thansk for reading!
UPC

**Comment by hhrutter** [comment]
v0.4.0 lets you turn individual form fields into read-only mode.
As of now rasterizing form fields & appearances is out of scope.

**Comment by uat230214** [comment]
@hhrutter 
Thanks for explaining. Yes, rasterizing certainly is the term to use.
In the mean time I found a way to do it using pd2topng.exe (from xpdf) that outputs png for each page, then joining them back with img2pdf.exe (from @josch at github).
I am not sure that png is the best solution in size/resolution ratio or not being size-efficient, or being there better solutions (even perhaps loseless) but it is what I found...

pdfcpu is the best I found in terms of text watermarking/overlaying (althought in the way yet to be able to overlay "ñ" and others from Latin alphabet.

Thanks for all!
